taskRequests/details/script.js

```diff
@@ -66,6 +68,16 @@ function renderTaskRequestDetails(taskRequest) {
   );
 }
 
+function updateStatus(status) {
+  const statusText = document.getElementById('taskRequest__status_text');
+  statusText.classList = [];
+  statusText.classList.add('taskRequest__status__chip');
+  statusText.classList.add(
+    `taskRequest__status__chip--${status?.toLowerCase()}`,
+  );
```

Comment:
Both classes can be added in single line as comma separated values.

Reply:
Isn't this more readable?

Reply:
Single line would be easier to understand both classes are being added to same element.
But sure! Current one is more readable considering large class names.